### ISSUE TITLE
[Balance]: Napalm is too powerful

### ISSUE THREAD

**Issue by Tiny-DM** [official-antistasi-community/A3-Antistasi#3391]
Napalm strikes are an AOE timed effect that's moderately effective against infantry and substantially more effective against vehicles, killing the crew and all systems while leaving the vehicles intact for recovery. More napalm airstrikes are being used to neutralize entire armored columns for cheap. A few possible solutions:

- Cap airstrike points (maybe war level and increase regeneration speed?)
- Adjust the cost of specific strikes. Everything costs 1 point right now, it could be changed to something along the lines of [Cluster, HE, Napalm to [1,2,3] or [1,3,5] to encourage usage of clusters at a risk of having a few loose toe poppers, and HE for softer vehicles and complete destruction
- Napalm should either leave vehicles functional or destroyed, not alive but ineffective
- Airstrikes should spawn from the friendly airfield so the ETA will be a couple minutes
- A combination of the above

**Comment by ante185** [comment]
If you make airstrikes more expensive, imma add the option to have a farmer hurl hand grenades out their plane

**Comment by Callum511** [comment]
Could reduce the number of bombs to 2 or 3. They are very strong and even a single direct hit on a small outpost will totally wipe it.

**Comment by ante185** [comment]
Rebel napalm strikes should probably be outright removed.
It's a super strong offensive strike with next to no downsides or consequences.

I'm also not sure where the threat/support sent in response to rebel airstrikes actually go anywhere useful

**Comment by Play3rgg** [comment]
I agree with ante, napalms not being a viable option is an exception, not the norm.

**Comment by Tiny-DM** [comment]
Probably fixed enough in #3608
